bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -66,6 +67,7 @@
 		<xs:attribute name="required" type="xs:boolean" />
 		<xs:attribute name="readOnly" type="xs:boolean" />
 		<xs:attribute name="multiple" type="xs:boolean" />
+		<xs:attribute name="unit" type="config-description:unitType" />
```

Comment:
We could put a file unit-maintenance.txt in the org.eclipse.smarthome.config.xml bundle. In the file I could point to the locations for maintaining the units list.